test/tests.bats

```diff
@@ -1208,10 +1216,9 @@ load 'test_helper/bats-assert/load'
 }
 
 @test "checking setup.sh: setup.sh email update" {
-  initialpass=$(cat ./config/postfix-accounts.cf | grep lorem@impsum.org | awk -F '|' '{print $2}')
-  run ./setup.sh email update lorem@impsum.org my password
-  sleep 10
-  updatepass=$(cat ./config/postfix-accounts.cf | grep lorem@impsum.org | awk -F '|' '{print $2}')
+  ./setup.sh -c mail email add lorem@impsum.org test_test && initialpass=$(cat ./test/config/postfix-accounts.cf | grep lorem@impsum.org | awk -F '|' '{print $2}')
```

Comment:
Did you check that initialpass is not empty? I had timing problems with it before. Also `-c mail` doesn't have an effect as the `addmailuser` script is running on a temporary container.

Reply:
Restarted the build